src/consumer/component/dependencies/dependency-resolver/dependencies-resolver.ts

```diff
@@ -696,32 +704,32 @@ either, use the ignore file syntax or change the require statement to have a mod
   processBits(originFile: PathLinuxRelative, fileType: FileType) {
     const bits = this.tree[originFile].bits;
     if (!bits || R.isEmpty(bits)) return;
-    // @ts-ignore AUTO-ADDED-AFTER-MIGRATION-PLEASE-FIX!
+    let componentId;
     bits.forEach(bitDep => {
-      const componentId: BitId = this.consumer.getComponentIdFromNodeModulesPath(bitDep, this.component.bindingPrefix);
-      if (this.overridesDependencies.shouldIgnoreComponent(componentId, fileType)) return;
-      const getExistingId = (): BitId | null | undefined => {
+      const version = bitDep.concreteVersion || bitDep.versionUsedByDependent;
+      if (bitDep.componentId) {
+        componentId = bitDep.componentId;
+      } else if (bitDep.fullPath) {
+        componentId = this.consumer.getComponentIdFromNodeModulesPath(bitDep.fullPath, this.component.bindingPrefix);
+      } else {
+        componentId = packageNameToComponentId(this.consumer, bitDep.name, this.component.bindingPrefix);
+      }
```

Comment:
add line spaces when you this kind of structure
instead of:
```js
if () {}
else if(){}
else if(){}
if() {}
```
do 
```
if () {}
else if(){}
else if(){}

if(){} 
```